test/core/Safe.ModuleManager.spec.ts

```diff
@@ -374,4 +375,305 @@ describe("ModuleManager", () => {
             expect(await safe.getModulesPaginated(AddressOne, 10)).to.be.deep.equal([[], AddressOne]);
         });
     });
+
+    describe("Module Guard", () => {
```

Comment:
Nit: Safe test are organized by `ContractName > functionName`. I'm not a fan of breaking that standard for module guards in this case. I.e. some of these tests should be moved to `execTransactionFromModuleReturnData` block for example.

Reply:
here: https://github.com/safe-global/safe-smart-account/pull/758/commits/6d34f6a1202823eb58063eb67c1cdb6c854ce32e